providers/softlayer/src/test/java/org/jclouds/softlayer/features/VirtualGuestApiLiveTest.java

```diff
@@ -133,7 +134,7 @@ public void testSetTagsOnVirtualGuest() throws Exception {
       Set<TagReference> tagReferences = found.getTagReferences();
       assertNotNull(tagReferences);
       for (String tag : tags) {
-         Iterables.contains(tagReferences, tag);
+         assertThat(Iterables.contains(tagReferences, tag)).isTrue();
```

Comment:
This is due to Guava 20 introducing annotating new methods which error-prone catches.